test/passes/vacuum_all-features.wast

```diff
@@ -795,3 +795,32 @@
   )
  )
 )
+
+(module
+  (event $e (attr 0) (param i32))
+  ;; When try body does not throw, try-body can be replaced with the try body
+  (func $try-test
+    (try
+      (drop (i32.const 0))
+      (catch
+        (drop (exnref.pop))
+      )
+    )
+  )
+
+  ;; The exception thrown in the inner try is caught by the inner catch, so the
+  ;; outer try body does not throw and the outer try-catch can be removed
+  (func $inner-try-test
+    (try
+      (try
+        (throw $e (i32.const 0))
+        (catch
+          (drop (exnref.pop))
+        )
+      )
```

Comment:
the catch blocks are identical, so it's not immediately obvious which is being removed. perhaps add some extra instruction to one of them, to differentiate them? (like `(drop (i32.const 1))` but I don't remember if this pass would remove it automatically; may need `(local.set (i32.const 1))`)

Reply:
Done.